src/app/config/app.config.ts

```diff
@@ -69,7 +69,10 @@ export class AppConfig {
           throw Error('Gb-backend URL is unspecified in the configuration.')
         }
         case 'fractalis-url': {
-          throw Error('Fractalis URL is unspecified in the configuration.')
+          if (this.getConfig('enable-fractalis-analysis') === true) {
+            throw Error('Fractalis URL is unspecified in the configuration.')
+          }
+          break;
```

Comment:
the control flow is a bit confusing here... in this case, break means returning null at the very end, right? we could as well return null here.